flow/scripts/cts.tcl

```diff
@@ -80,7 +80,12 @@ if { [info exists ::env(SKIP_PIN_SWAP)] } {
   append additional_args " -skip_pin_swap"
 }
 
-repair_timing -skip_gate_cloning {*}$additional_args
+if { [info exists ::env(ENABLE_GATE_CLONING)] } {
```

Comment:
Please update [FlowVariables.md](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/docs/user/FlowVariables.md) with description for `ENABLE_GATE_CLONING` along with this PR.

Reply:
Done. Thank you for the reminder :+1: